packages/@uppy/webcam/src/Webcam.jsx

```diff
@@ -588,6 +590,13 @@ export default class Webcam extends UIPlugin {
   }
 
   install () {
+    if (this.opts.mobileNativeCamera && isMobile()) {
```

Comment:
I think folks should be able to decide if they want to force the native camera or not, and they should be the one using `is-mobile`. Having `is-mobile` as a dependency will create all kinds of problems if we forget to keep it up-to-date, and can be very frustrating for our users if they have no way of forcing the native camera even in development.